python/pyspark/sql/connect/client.py

```diff
@@ -36,6 +41,49 @@
 )
 
 
+def _configure_logging() -> logging.Logger:
+    """Configure logging for the Spark Connect clients."""
+    logger = logging.getLogger("pyspark.sql.connect.client")
+    handler = logging.StreamHandler()
+    handler.setFormatter(
+        logging.Formatter(fmt="%(asctime)s %(process)d %(levelname)s %(funcName)s %(message)s")
+    )
+    logger.addHandler(handler)
+
+    # Check the environment variables for log levels:
+    level = os.getenv("SPARK_CONNECT_LOG_LEVEL", "none").lower()
```

Comment:
No biggie but we could do this like:

```python
if "SPARK_CONNECT_LOG_LEVEL" os.environ:
    logger.setLevel(os.getenv("SPARK_CONNECT_LOG_LEVEL").upper())
else:
    logger.disabled = True
```

so it throws an exception if a weird value is set.